core/src/test/java/hudson/FunctionsTest.java

```diff
@@ -562,4 +562,29 @@ synchronized Stack suppressed(Throwable... suppressed) {
             return this;
         }
     }
+
+    @Test
+    @PrepareForTest(Stapler.class)
+    @Issue("JENKINS-62419")
+    public void testLocaleForItalian() throws Exception {
+        String contextPath = "/jenkins";
+        StaplerRequest req = createMockRequest(contextPath);
+        mockStatic(Stapler.class);
+        when(req.getLocale()).thenReturn(Locale.ITALIAN);
+        String localeDisplayLanguage = Functions.getLocale(req);
+        assertEquals("italiano", localeDisplayLanguage);
```

Comment:
As it's `English` in the other test, shouldn't it be `Italiano` here with a upper-case `i` no?

Reply:
Hi,
I thought the same thing, checked the code with debug, It returns italiano.
![jenkins-locale-test-output-2](https://user-images.githubusercontent.com/17325506/86059054-48d98e00-ba62-11ea-8868-0abb0846a409.png)
![jenkins-locale-test-output](https://user-images.githubusercontent.com/17325506/86059055-49722480-ba62-11ea-9af9-acf38e3d3a6c.png)

Reply:
Ok, no problem. I just thought it was odd.